app/src/main/java/com/sopetit/softie/ui/onboarding/splash/SplashActivity.kt

```diff
@@ -0,0 +1,54 @@
+package com.sopetit.softie.ui.onboarding.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import androidx.fragment.app.Fragment
+import com.sopetit.softie.R
+import com.sopetit.softie.databinding.ActivitySplashBinding
+import com.sopetit.softie.ui.storytelling.StoryTellingActivity
+import com.sopetit.softie.util.binding.BindingActivity
+
+class SplashActivity : BindingActivity<ActivitySplashBinding>(R.layout.activity_splash) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        initCreateRandomVersion()
+    }
+
+    private fun initCreateRandomVersion() {
+        val versionList: List<Int> = listOf(1, 2, 3, 4)
+
+        when (versionList.random()) {
```

Comment:
```suggestion
        when (Random.nextInt(1, 5)) {
```
코드 일케 줄일수 있을거 같네요 ㅎㅎ